### ISSUE TITLE
Video streaming problems in several apps

### ISSUE THREAD

**Issue by BabyDomo** [MatthewBooth/JFLTE-GPE#38]
Netflix, Youtube and Facebook Thumbnails, lag and stutter after jumping from 5.0.1 to 5.1. The only workaround I have found useful is to press "back" button and then try again loading desired video.

**Comment by BabyDomo** [comment]
A special app (Axtel Conmigo) from a mexican ISP called Axtel, whose app is used to be able to receive calls when out of home, or to make calls from wifi connection using your own Axtel account instead of your plan, used to work just fine at 5.0.1. Now the app just won't let you even register your mobil phone (which is a pre requisite for starting using the app). After placing my phone number, the app just launches an error that "couldn't register number." The app is called "Axtel Conmigo"

**Comment by drew5494** [comment]
Yes I have seen this too in youtube but even in gaming, like asphalt 8 will continuously freeze
